dashboard/app/helpers/levels_helper.rb

```diff
@@ -105,10 +105,11 @@ def app_options
     # Provide the channel for templated and applab levels.
     set_channel if @level.project_template_level || @level.game == Game.applab
 
+    callouts = params[:share] ? [] : select_and_remember_callouts(params[:show_callouts])
     # Set videos and callouts.
     view_options(
       autoplay_video: select_and_track_autoplay_video,
-      callouts: select_and_remember_callouts(params[:show_callouts])
+      callouts: callouts
```

Comment:
Setting callouts to [] in projects#show didn't seem to be sufficient. Are there ever going to be times where we're sharing but still want to see callouts?